src/sentry/grouping/strategies/message.py

```diff
@@ -242,7 +259,7 @@ def _handle_match(match: Match[str]) -> str:
                 6424467,
             ]
```

Comment:
At the end of the experiment, this should be removed.
In general, we don't do this but use an option to declare projects. See this example: https://github.com/getsentry/sentry-options-automator/blob/47947e8257edb05fe778fa9d6b0b42c4096c8411/options/regions/us/app.yaml#L222